Makefile

```
@@ -41,14 +41,15 @@ $(TOOLS)/kubectl: $(TOOLS)
 .PHONY: tools
 tools: $(JQ) $(YQ) $(KUBECTL)
 
-OTEL_VERSION?=$(COLLECTOR_CONTRIB_VERSION)
+OTEL_COLLECTOR_VERSION?=$(GBOC_VERSION)
```

Nit: by convention this is usually the following.

```suggestion
OTEL_COLLECTOR_VERSION ?= $(GBOC_VERSION)
```